<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>basic_mode</class>
 <widget class="QWidget" name="basic_mode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>欢乐连连看--基础模式</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>570</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="BTN_START">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>开始游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_HELP">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>490</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_PAUSE">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>暂停游戏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_TIP">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>130</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_REARRANGE">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>70</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>重排</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BTN_SETTING">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>430</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="CountDownBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>641</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
    <property name="value">
     <number>300</number>
    </property>
    <property name="format">
     <string>%vs</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
